src/components/WorkOrder/Photos/hooks/useFileUpload.js

```diff
@@ -0,0 +1,182 @@
+import { useState } from 'react'
+import { frontEndApiRequest } from '@/utils/frontEndApiClient/requests'
+
+import axios from 'axios'
+import { filesize } from 'filesize'
+
+const useFileUpload = (workOrderReference, onSuccess) => {
+  const [isUploading, setIsUploading] = useState(false)
+  const [validationError, setValidationError] = useState(null)
+  const [requestError, setRequestError] = useState(null)
+  const [uploadSuccess, setUploadSuccess] = useState(null)
+
+  const [files, setFiles] = useState([])
+
+  const allowedFileTypes = new Set(['image/png', 'image/jpeg'])
+  const MAX_FILE_SIZE = 20000000
```

Comment:
If it matters, it's not 20MB

Reply:
not really. this is arbitrary and intented to be too high